Main.py

```diff
@@ -0,0 +1,117 @@
+import pygame, sys, time, math, random
+from pygame.locals import *
+
+pygame.init()  #  Initiates Pygame
+img1 = pygame.image.load('BigSmoke.jpg')
+img2 = pygame.image.load('Harambe.jpg')
+img3 = pygame.image.load('Doge.jpg')
+img4 = pygame.image.load('YeaBoi.jpg')
+img5 = pygame.image.load('Dio.jpg')
+img6 = pygame.image.load('Jaguars.jpg')
+img7 = pygame.image.load('Issac.jpg')
+myfont = pygame.font.SysFont("comic", 100) #Imports text font
```

Comment:
Try writing comments to explain blocks of code rather than lots of inline explanations.